cmd/state-update-dialog/main.go

```diff
@@ -46,8 +46,8 @@ func main() {
 		exitCode = 1
 		return
 	}
-	logging.CurrentHandler().SetConfig(cfg)
 	rollbar.SetupRollbar(constants.StateTrayRollbarToken) // We're using the state tray project cause it's closely related
+	rollbar.SetConfig(cfg)
```

Comment:
Can we amend `SetupRollbar` instead?

Reply:
I tried this initially, but there are places where we want to setup rollbar first, then load the config. If there's an error setting up the config, we want to report that to rollbar. If we wait to setup rollbar until after loading config, and that load fails, we'll never know about it.